kinto/__main__.py

```diff
@@ -30,6 +30,8 @@ def main(args=None):
                         dest='backend',
                         required=False,
                         default=None)
+    parser.add_argument('--version', action='version', version=__version__,
+                            help='Print the Kinto version and exit.')
```

Comment:
flake8 reports that ``kinto/__main__.py:34:29: E127 continuation line over-indented for visual indent``